library/cartridge_edit_topology.py

```diff
@@ -320,20 +322,113 @@ def get_edit_failover_priority_params(replicasets, cluster_replicasets, cluster_
             continue
 
         if cluster_replicaset['instances'][:len(failover_priority)] != failover_priority:
-            failover_priority_uuids = [
-                cluster_instances[instance_name]['uuid'] for instance_name in failover_priority
-                if instance_name in cluster_instances  # false if instance is expelled
-            ]
+            failover_priority_uuids = []
+            for instance_name in failover_priority:
+                if instance_name not in cluster_instances or not cluster_instances[instance_name].get('uuid'):
+                    return None, "Instance %s from %s failover_priority isn't joined to cluster" % (
+                        instance_name, alias
+                    )
+                failover_priority_uuids.append(cluster_instances[instance_name]['uuid'])
 
             edit_replicasets_params.append({
                 'uuid': cluster_replicaset['uuid'],
                 'failover_priority': failover_priority_uuids,
             })
 
+    return edit_replicasets_params, None
+
+
+def add_edit_server_param_if_required(edit_server_params, instance_params, cluster_instance, param_name):
```

Comment:
```suggestion
def add_param_to_edit_server_params_if_required(edit_server_params, instance_params, cluster_instance, param_name):
```

Reply:
What's the difference?) Maybe `append_edit_server_params_if_required`?

Reply:
There are two verbs in a name here. The name gives the impression that the function is editing something, but it works with parameters for editing